### DISCUSSION TITLE
Add "Original estimate" field to metadata when creating an issue

### DISCUSSION THREAD

**Discussion by Mamenzul** [ankitpokhrel/jira-cli#354] [Ideas]
Hello, fist, I'm very grateful for all the work you've done on this projet !!

Maybe I'm missing something but I don't see a way to update nor input "Original estimate" value when creating an issue. Was it a choice to not add it ?

**Comment by freemo** [comment]
Any update on this? I have been trying for a day to add this to the config as a "custom" field and getting no where... It would be great if someone answered to indicate if this is even possible.

I have to wonder why the code allows custom fields but not built-in fields. There is probably a reason but seems like if all the built-in fields arent supported then just allow you to define them as custom fields maybe?

**Comment by gsweats** [comment]
Anybody ever find anything on this...almost a year later and I have the same issue :)

**Comment by ankitpokhrel** [comment]
This should already be possible since v1.5.2 & above
